deploy.sh

```diff
@@ -20,6 +20,15 @@ if [[ -z "$SVN_PASSWORD" ]]; then
 	exit 1
 fi
 
+# Check if the DRY_RUN ENV variable is set, otherwise set it to false.
+if [[ -z "$DRY_RUN" ]]; then
+	DRY_RUN=false;
+fi
+
+if [[ "$DRY_RUN" == true ]]; then
+	echo "ℹ︎ DRY_RUN is true"
```

Comment:
As an input, this might be more useful as more of an informational message since the consumer won't be providing the env directly anymore.
```suggestion
	echo "ℹ︎ Dry run: no files will be committed to subversion."
```